src/app/main/component/user/components/profile/edit-profile/edit-profile.component.ts

```diff
@@ -53,7 +53,7 @@ export class EditProfileComponent extends FormBaseComponent implements OnInit, O
   public checkEcoPlaces = false;
   public checkShoppingList = false;
   public namePattern = /^(?!\.)(?!.*\.$)(?!.*?\.\.)[a-zA-Zа-яА-Я0-9.]{6,30}$/;
-  public cityPattern = /^[a-zA-Zа-яА-Я][a-zA-Zа-яА-Я!\-\,’)( ]*$/;
+  public cityPattern = /^[іІєЄїЇёЁa-zA-Zа-яА-Я][іІєЄїЇёЁa-zA-Zа-яА-Я!\-,’)( ]*$/;
```

Comment:
Do we really need '!' this char in the city regex?

Reply:
not really. gonna change it